CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## - Unreleased
+
+- Updated welcome message and links to Terms of Service and Privacy Policy.
```

Comment:
As per `https://keepachangelog.com/en/1.0.0/` types of changes
```suggestion
- Changed welcome message and links to Terms of Service and Privacy Policy.
```

Reply:
done